src/plugins/discover/public/application/context/context_app.tsx

```diff
@@ -30,7 +30,7 @@ import { DocViewFilterFn } from '../../services/doc_views/doc_views_types';
 const ContextAppContentMemoized = memo(ContextAppContent);
 
 export interface ContextAppProps {
-  indexPattern: IndexPattern;
+  indexPattern: DataView;
```

Comment:
how do we feel about renaming props to `dataView`?

Reply:
for sure we should do this ... in a follow up :) 👍